src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export type User = {
```

Comment:
Consider using interface instead of type for better compatibility with the TypeScript language features such as declaration merging.